.github/workflows/build-and-release-to-spin.yml

```diff
@@ -63,6 +64,14 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      # Reference: https://docs.docker.com/build/ci/github-actions/push-multi-registries/
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
```

Comment:
When the GHA workflow ran as written here, GHCR returned an HTTP 403 Forbidden response.

I got this `github.*` value from the Docker docs. I actually use a different `github.*` value on another project. I will try that instead.

Reply:
For example, I set up GHCR deployment in the `nmdc-edge` repo (I've jumped in to do some containerization of that project). There, I configured the `login-action` step this way:

```yaml
            - name: Authenticate with container registry
              uses: docker/login-action@v3
              with:
                  registry: ghcr.io
                  username: ${{ github.actor }}           # <--
                  password: ${{ secrets.GITHUB_TOKEN }}
```

I'll try replicating that in this repo:

```diff
- username: ${{ github.repository_owner }}
+ username: ${{ github.actor }}
```